kimchi/js/bindings.js

```diff
@@ -1149,8 +1149,10 @@ var caml_plonk_gate_of_rust = function (gate) {
 // Provides: caml_fp_plonk_gate_to_rust
 // Requires: plonk_wasm, caml_plonk_wires_to_rust, caml_u8array_vector_to_rust_flat_vector
 var caml_fp_plonk_gate_to_rust = function (gate) {
+    var gateType = gate[1];
+    if (gateType > 6) gateType = gateType + 4;
     return new plonk_wasm.WasmFpGate(
-        gate[1],
+        gateType,
```

Comment:
this is a temportary hack until the following proof systems fix is upstreamed to this PR: https://github.com/o1-labs/proof-systems/pull/1091

Reply:
reverted!